typescript/infra/config/environments/mainnet2/agent.ts

```diff
@@ -72,6 +72,8 @@ export const hyperlane: AgentConfig<MainnetChains> = {
       gasPaymentEnforcementPolicy: {
         type: GasPaymentEnforcementPolicyType.None,
       },
+      transactionGasLimit: undefined,
+      skipTransactionGasLimitFor: [42161],
```

Comment:
Could we do this?
```
import { chainMetadata } from '@hyperlane-xyz/sdk';
// ...
   skipTransactionGasLimitFor: [chainMetadata.arbitrum.id],
```
Similar idea for testnet3 too

Also a quick comment on why we are skipping for arbitrum would be nice